PRChecker/Shared/Common Views/TagListView.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  TagListView.swift
+//  PRChecker
+//
+//  Created by Chen Yuhan on 2021/11/14.
+//
+
+import SwiftUI
+
+struct TagListView: View {
+    var tagViews: [TagView]
+
+    @State private var totalHeight: CGFloat = 0
+
+    // Ref: https://stackoverflow.com/questions/62102647/swiftui-hstack-with-wrap-and-dynamic-height
+    var body: some View {
+        var width: CGFloat = 0
+        var height: CGFloat = 0
+
+        return VStack {
```

Comment:
Is the return because of the addition of `width` and `height`?

Reply:
In fact this `return` is not needed